test/testpartner.ts

```diff
@@ -0,0 +1,99 @@
+import {
+  addEdgeCorePlugins,
+  EdgeSwapRequest,
+  lockEdgeCorePlugins,
+  makeFakeEdgeWorld
+} from 'edge-core-js'
+
+import edgeExchangePlugins from '../src'
+import { btcCurrencyInfo } from './fakeBtcInfo'
+import { makeFakePlugin } from './fakeCurrencyPlugin'
+import { ethCurrencyInfo } from './fakeEthInfo'
+
+async function main(): Promise<void> {
+  const allPlugins = {
+    bitcoin: makeFakePlugin(btcCurrencyInfo),
```

Comment:
This part is nice, injecting the currencyInfo like this.

Reply:
Thx